tests/utils/useAtomCallback.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { fireEvent, render } from '@testing-library/react'
+import { Provider, useAtom, atom } from '../../src/index'
+import { useAtomCallback } from '../../src/utils'
+
+it('useAtomCallback with get', async () => {
+  const countAtom = atom(0)
+
+  const Parent: React.FC = () => {
+    const [count, setCount] = useAtom(countAtom)
```

Comment:
It would be better, we we didn't have this here, because that should be the purpose of useAtomCallback.

Reply:
Sorry, which line you mean?

Reply:
L10 `useAtom(countAtom)` wouldn't be in `Parent`.